### DISCUSSION TITLE
Android 12 compatibility

### DISCUSSION THREAD

**Discussion by donsaimon** [BimmerGestalt/AAIdrive#489] [General]
Hi, is the app compatible with Android 12?
I am using it with Android 11 on Samsung Z Flip 3, but I am not sure if the app will work again if I update to Android 12...

**Comment by hufman** [comment]
Yes, the Sentry analytics (as of today) say 10% of car connections are with Android 12.

**Comment by donsaimon** [comment]
Thank you. And did you have some return from Android 12 users?

Da: ***@***.***
Inviato: 6 Gennaio 2022 16:53
A: ***@***.***
Rispondi a: ***@***.***
Cc: ***@***.***; ***@***.***
Oggetto: Re: [BimmerGestalt/AAIdrive] Android 12 compatibility (Discussion #489)


Yes, the Sentry analytics (as of today) say 10% of car connections are with Android 12.

—
Reply to this email directly, view it on GitHub<https://github.com/BimmerGestalt/AAIdrive/discussions/489#discussioncomment-1919461>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ASMCOD4A562P6GGQGW4XQB3UUW3IZANCNFSM5LMSXMZQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by hufman** [reply]
I'm not sure I understand you, what is "return from Android 12 users"?

**Comment by donsaimon** [comment]
Sorry I mean feedbacks

Da: ***@***.***
Inviato: 6 Gennaio 2022 17:43
A: ***@***.***
Rispondi a: ***@***.***
Cc: ***@***.***; ***@***.***
Oggetto: Re: [BimmerGestalt/AAIdrive] Android 12 compatibility (Discussion #489)


I'm not sure I understand you, what is "return from Android 12 users"?

—
Reply to this email directly, view it on GitHub<https://github.com/BimmerGestalt/AAIdrive/discussions/489#discussioncomment-1919797>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ASMCOD7AK6UKODKB4I45BQLUUXBEHANCNFSM5LMSXMZQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by hufman** [reply]
The only issue I've heard of so far is #456, that Android 11 and 12 sometimes blocks the app icons in the Notification view. The user otherwise had no troubles with the rest of the app on their Android 12 phone.

**Comment by donsaimon** [comment]
Ok thank you

Da: ***@***.***
Inviato: 6 Gennaio 2022 18:12
A: ***@***.***
Rispondi a: ***@***.***
Cc: ***@***.***; ***@***.***
Oggetto: Re: [BimmerGestalt/AAIdrive] Android 12 compatibility (Discussion #489)


The only issue I've heard of so far is #456<https://github.com/BimmerGestalt/AAIdrive/issues/456>, that Android 11 and 12 sometimes blocks the app icons in the Notification view. The user otherwise had no troubles with the rest of the app on their Android 12 phone.

—
Reply to this email directly, view it on GitHub<https://github.com/BimmerGestalt/AAIdrive/discussions/489#discussioncomment-1919974>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ASMCOD3CIQCYXCKCSULLHTLUUXERJANCNFSM5LMSXMZQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by Ringorocks25** [reply]
I see you have a flip 3. I Also have the flip 3. I'm having an issue (Android 12) where while in Spotify after the phone screen times out and goes off,  30 sec after that it freezes the screen on the head unit (2019 Cooper S).   I have to wake the phone up to get it to start the screen and song time back.  Have you experienced this or does anyone know what causes this?  I assume it's a setting somewhere but I'm very savvy and haven't found it. Is it on the phone,  the app, or Spotify? Thanks

**Reply by hufman** [reply]
This sounds like the phone is applying some battery saving options to either MyBMW or AAIdrive. Try out some of these [Samsung-specific settings](https://dontkillmyapp.com/samsung#android-11) to see if they help! Unfortunately I haven't found any other methods to work around this, the user has to manually grant this exception to each app that needs it.

**Reply by Ringorocks25** [reply]
Sorry I hate it when no one checks back in and I was almost that guy. Yes I had power saving turned off on everything but....AAIDrive. Shame on me. The app is awesome and works flawlessly. Really appreciate all the hard work that went into this. Now if we could just get Waze integrated I think a lot of us would be happy campers. I did do the mirrored screen with Waze on but its pointless to have the phone on and open in the seat next to you while its projecting to the dash screen. Not for nothing it was really cool to see Waze on the Infotainment unit.

**Comment by donsaimon** [comment]
Hi, no issue here. My phone's screen is normally off when I am in the car. Spotify correctly works. Even if I manually wake the screen up while I'm the car, no issue when the screen times out and goes off...
Ciao
Simone

Da: ***@***.***
Inviato: 15 Gennaio 2022 19:06
A: ***@***.***
Rispondi a: ***@***.***
Cc: ***@***.***; ***@***.***
Oggetto: Re: [BimmerGestalt/AAIdrive] Android 12 compatibility (Discussion #489)


I see you have a flip 3. I Also have the flip 3. I'm having an issue (Android 12) where while in Spotify after the phone screen times out and goes off, 30 sec after that it freezes the screen on the head unit (2019 Cooper S). I have to wake the phone up to get it to start the screen and song time back. Have you experienced this or does anyone know what causes this? I assume it's a setting somewhere but I'm very savvy and haven't found it. Is it on the phone, the app, or Spotify? Thanks

—
Reply to this email directly, view it on GitHub<https://github.com/BimmerGestalt/AAIdrive/discussions/489#discussioncomment-1975490>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ASMCODZ2CA3DT6OFI6B5IBTUWGZR3ANCNFSM5LMSXMZQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by patalmqx** [comment]
I am about to invest 300 bucks for Apps coding into my NBT. The only question I have: Can I be sure or almost sure that my new Galaxy A52 works with AAIdrive? My previous Huawei p10 lite didn't work at all (no accessory mode?). Can you readthis out? I guess Samsung allows such connections? Thanks!

**Reply by hufman** [reply]
I of course can't guarantee anything, but the Sentry analytics shows that Galaxy A52 5G model phones have connected successfully to a car 355 times since 2021-11-04, making up 0.54% of the total connections. I don't see any regular A52 or A52s devices, but I presume they would have equal success because they are running the same version of Android and OneUI. :+1:

**Comment by patalmqx** [comment]
Great input thanks!!

Walter Huf ***@***.***> schrieb am Mi., 2. Feb. 2022, 01:58:
